commerce-account-item-selector-web/src/main/java/com/liferay/commerce/account/item/selector/web/internal/display/context/CommerceAccountItemSelectorViewDisplayContext.java

```diff
@@ -66,6 +66,12 @@ public String getOrderByType() {
 	}
 
 	public PortletURL getPortletURL() {
+		_portletURL.setParameter(
+			"checkedCommerceAccountIds",
+			ParamUtil.getString(
+				_commerceAccountItemSelectorRequestHelper.getRenderRequest(),
+				"checkedCommerceAccountIds"));
+
```

Comment:
@alecsloan it makes more sense to me to put this in the constructor, so it runs once, rather than re-setting this parameter every time we get the PortletURL. What do you think?

Reply:
I agree, good catch!